<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroduceForm</class>
 <widget class="QWidget" name="IntroduceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetLeft" native="true">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>675</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetLeft
{
	border-image: url(:/resource/images/common/Transparent_White_80.png);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QWidget" name="widgetList" native="true">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="listWidget">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="widgetDown" native="true">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>0</y>
           <width>120</width>
           <height>12</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/resource/images/bb_classify/roll_down.png);</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QWidget" name="widgetBottom" native="true">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>76</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>76</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="widgetLogo" native="true">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#label
{
	border-image: url(:/resource/images/common/Transparent_White_70.png);
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widgetUp" native="true">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>120</width>
           <height>12</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/resource/images/bb_classify/roll_up.png);</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="IntroduceFromRight" name="widgetShowIntroduce" native="true">
     <property name="minimumSize">
      <size>
       <width>845</width>
       <height>675</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetShowIntroduce
{
	background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widgetMid" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>675</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>235</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/resource/images/bb_classify/blackwithblueline.png);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widgetRight" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widgetTitle" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>531</width>
             <height>51</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>466</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonClose">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>43</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>43</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/resource/images/common/Back_black.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget" native="true">
           <widget class="QLabel" name="labelText">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>240</y>
              <width>471</width>
              <height>301</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="labelImage">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>261</width>
              <height>171</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntroduceFromRight</class>
   <extends>QWidget</extends>
   <header>IntroduceFromRight.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
